### ISSUE TITLE
Upgrade to 6.1.0 stalls after minimal applied

### ISSUE THREAD

**Issue by jjlawren** [reloxx13/TasmoAdmin#165]
TasmoAdmin console:
```
[06-07-2018 15:19:42][GLOBAL] Start update process
[06-07-2018 15:19:42][ID-1][STEP-1][GLOBAL] Start Step 1
[06-07-2018 15:19:42][ID-1][STEP-1][CHECK] Check Connection | TRY => 1
[06-07-2018 15:19:42][ID-1][STEP-1][CHECK] Connection Successfully! | Current Version => 5.14.0
[06-07-2018 15:19:42][ID-1][STEP-1][OTAURL] Set OTA URL for Firmware => MINIMAL
[06-07-2018 15:19:42][ID-1][STEP-1][OTAURL] OTA Url => http://10.10.10.1:80/data/firmwares/sonoff-minimal.bin
[06-07-2018 15:19:43][ID-1][STEP-1][OTAURL] OTA URL set for Firmware => MINIMAL
[06-07-2018 15:19:43][ID-1][STEP-1][UPDATE] Trigger device update...
[06-07-2018 15:19:43][ID-1][STEP-1][UPDATE] Device update triggered
[06-07-2018 15:19:43][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 60
[06-07-2018 15:20:43][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:20:43][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:21:13][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:21:13][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:21:43][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:21:43][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:22:13][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:22:13][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:22:43][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:22:43][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:23:14][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:23:14][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
[06-07-2018 15:23:44][ID-1][STEP-1][CHECK UPDATE] Update not done now
[06-07-2018 15:23:44][ID-1][STEP-1][CHECK UPDATE] Wait for update | Seconds => 30
```
Sonoff MQTT messages:
```
stat/sonoff/RESULT {"OtaUrl":"http://10.10.10.1:80/data/firmwares/sonoff-minimal.bin"}
stat/sonoff/RESULT {"Upgrade":"Version 5.14.0 from http://10.10.10.1:80/data/firmwares/sonoff-minimal.bin"}
tele/sonoff/LWT Offline
tele/sonoff/LWT Online
cmnd/sonoff/POWER (null)
stat/sonoff/UPGRADE Successful. Restarting
tele/sonoff/LWT Online
cmnd/sonoff/POWER (null)
tele/sonoff/INFO1 {"Module":"Sonoff Pow","Version":"6.1.0-minimal","FallbackTopic":"DVES_AAAAAA","GroupTopic":"sonoffs"}
tele/sonoff/INFO2 {"WebServerMode":"Admin","Hostname":"sonoff-1234","IPAddress":"10.10.10.10"}
tele/sonoff/INFO3 {"RestartReason":"Software/System restart"}
stat/sonoff/RESULT {"POWER1":"ON"}
stat/sonoff/POWER1 ON
tele/sonoff/STATE {"Time":"2018-07-06T22:20:30","Uptime":"0T00:00:13","Vcc":3.242,"POWER1":"ON","Wifi":{"AP":1,"SSId":"ssid","RSSI":46,"APMac":"AA:AA:AA:AA:AA:AA"}}
tele/sonoff/STATE {"Time":"2018-07-06T22:25:30","Uptime":"0T00:05:13","Vcc":3.241,"POWER1":"ON","Wifi":{"AP":1,"SSId":"ssid","RSSI":44,"APMac":"AA:AA:AA:AA:AA:AA"}}
```
The previous upgrade from 5.12->5.14.0 went fine via TasmoAdmin. The Sonoff device's minimal web interface is available.

**Comment by arendst** [comment]
I guess the problem is with me undefining http access to the minimal firmware image to save space AND the way tasmoadmin wants to upgrade a device.

Normal upgrade process by using either mqtt, serial or web interface works just fine. The problem is with the TasmoAdmin interface to sonoff-minimal.

**Comment by arendst** [comment]
To save people from running into this I'll make some changes to the minimal version tomorrow.

In the meantime just use the correct way as noted above ;-)

**Comment by HFeenstra** [comment]
thank you. 
BTW followed the executes you requested me to do:

- otaurl "http://localip:port/data/firmwares/sonoff.bin"
- upload 1

No success:
stat/sonoff/UPGRADE = Failed HTTP error: connection refused

Wait for tomorrow ;)

**Comment by arendst** [comment]
Just released the fix as v6.1.0 (again). wait a few minutes for the binaries to pop up.

**Comment by reloxx13** [comment]
this is fixed in v1.3.3

changed to OTA magic, but sadly i wont be able to show a status of teh progress, since i dont know what tasmota does while updating.

would be good if the upgrade 1 command already respond if the minimal is needed or not, but i dont know if this is working, depending on the ota magic progress.
